CONTRIBUTING.md

```diff
@@ -137,7 +137,7 @@ installed on your system.
 Run unit tests and coverage using tox for multiple Python versions:
 
 ```bash
-$ tox -e py36 -e py37
+$ tox -e py38 -e py39
```

Comment:
What about py37?

Reply:
Or py3.10?

Reply:
We are going to assume devs know about `tox` and how to use it.  I've updated the doc with that in mind.  Let me know what you think